odoo/addons/base/models/ir_ui_view.py

```diff
@@ -201,6 +201,23 @@ class IrUiView(models.Model):
          """)
     model_id = fields.Many2one("ir.model", string="Model of the view", compute='_compute_model_id', inverse='_inverse_compute_model_id')
 
+    invalid_xpaths_from_arch = fields.Json(compute='_compute_invalid_xpaths_from_arch')
+
+    @api.depends('arch', 'inherit_id')
+    def _compute_invalid_xpaths_from_arch(self):
+        """ Compute invalid XPaths for the current XML architecture when
+        the view is a studio view.
+        """
+        if self.xml_id.startswith('studio_customization') and self.inherit_id:
```

Comment:
compute takes a recordset with at least one record in it: if someone access the field with multiple views, this method will crash.
To be factorized as a loop